src/Apps/W1/Quality Management/app/src/Integration/Receiving/QltyReceivingIntegration.Codeunit.al

```diff
@@ -473,7 +473,7 @@ codeunit 20411 "Qlty. Receiving Integration"
     /// </summary>
     /// <param name="HasTest"></param>
     /// <param name="QltyInspectionTestHeader">The quality inspection test involved</param>
-    /// <param name="WarehouseJournalLine">VAR Record "Warehouse Journal Line".</param>
+    /// <param name="WarehouseJournalLine">var Record "Warehouse Journal Line".</param>
```

Comment:
This is not proper documentation. What is the parameters suppose to represent? Not the signature.